src/ElectricPokemon.java

```diff
@@ -0,0 +1,103 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
```

Comment:
Awesome, je hebt alle code geimplementeerd. Je hebt ook goed rekening gehouden met de naamgeving (die is namelijk belangrijk voor deze applicatie)